### ISSUE TITLE
Initializing a blank card with pkcs15-init on a  STARCOS Card Version 2.3

### ISSUE THREAD

**Issue by Nepitwin** [OpenSC/OpenSC#526]
I've got 2 STARCOS cards in Version 2.3. I try to create Blank Cards with the getting started guide on an Ubuntu 14.04.

First i erase my card with pkcs15-init -E and after all i tried to create an blank card with pkcs15-init -C.
```
pkcs15-init -C
Using reader with a card: REINER SCT cyberJack RFID komfort (5654335523) 00 00
New Security Officer PIN (Optional - press return for no PIN).
Please enter Security Officer PIN: 
Please type again to verify: 
Unblock Code for New User PIN (Optional - press return for no PIN).
Please enter User unblocking PIN (PUK): 
Please type again to verify: 

Failed to create PKCS #15 meta structure: Not allowed
```
Log
```
======================================================================
0x7fce68050740 15:03:48.662 [pkcs15-init] apdu.c:399:sc_single_transmit: returning with: 0 (Success)
0x7fce68050740 15:03:48.662 [pkcs15-init] apdu.c:552:sc_transmit: returning with: 0 (Success)
0x7fce68050740 15:03:48.662 [pkcs15-init] card.c:403:sc_unlock: called
0x7fce68050740 15:03:48.662 [pkcs15-init] card-starcos.c:1591:starcos_check_sw: sw1 = 0x69, sw2 = 0x85
0x7fce68050740 15:03:48.662 [pkcs15-init] iso7816.c:121:iso7816_check_sw: Conditions of use not satisfied
0x7fce68050740 15:03:48.662 [pkcs15-init] card.c:822:sc_card_ctl: returning with: -1209 (Not allowed)
0x7fce68050740 15:03:48.662 [pkcs15-init] pkcs15-lib.c:835:sc_pkcs15init_add_app: Create 'DIR' error: -1209 (Not allowed)
Failed to create PKCS #15 meta structure: Not allowed
0x7fce68050740 15:03:48.662 [pkcs15-init] pkcs15-lib.c:417:sc_pkcs15init_unbind: called
0x7fce68050740 15:03:48.662 [pkcs15-init] pkcs15-lib.c:418:sc_pkcs15init_unbind: Pksc15init Unbind: 0:(nil):1
0x7fce68050740 15:03:48.662 [pkcs15-init] card.c:403:sc_unlock: called
0x7fce68050740 15:03:48.662 [pkcs15-init] reader-pcsc.c:566:pcsc_unlock: called
0x7fce68050740 15:03:48.669 [pkcs15-init] card.c:305:sc_disconnect_card: called
0x7fce68050740 15:03:48.669 [pkcs15-init] reader-pcsc.c:512:pcsc_disconnect: called
0x7fce68050740 15:03:48.807 [pkcs15-init] card.c:326:sc_disconnect_card: returning with: 0 (Success)
0x7fce68050740 15:03:48.807 [pkcs15-init] ctx.c:815:sc_release_context: called
0x7fce68050740 15:03:48.807 [pkcs15-init] reader-pcsc.c:766:pcsc_finish: called
```

On my second Card i've got the following error
Failed to create PKCS #15 meta structure: Security status not satisfied

Log
```
======================================================================
0x7fc8ab586740 15:01:44.202 [pkcs15-init] apdu.c:399:sc_single_transmit: returning with: 0 (Success)
0x7fc8ab586740 15:01:44.202 [pkcs15-init] apdu.c:552:sc_transmit: returning with: 0 (Success)
0x7fc8ab586740 15:01:44.202 [pkcs15-init] card.c:403:sc_unlock: called
0x7fc8ab586740 15:01:44.202 [pkcs15-init] card-starcos.c:1591:starcos_check_sw: sw1 = 0x69, sw2 = 0x82
0x7fc8ab586740 15:01:44.202 [pkcs15-init] iso7816.c:121:iso7816_check_sw: Security status not satisfied
0x7fc8ab586740 15:01:44.202 [pkcs15-init] card.c:467:sc_create_file: returning with: -1211 (Security status not satisfied)
0x7fc8ab586740 15:01:44.202 [pkcs15-init] pkcs15-lib.c:3521:sc_pkcs15init_create_file: Create file failed: -1211 (Security status not satisfied)
0x7fc8ab586740 15:01:44.202 [pkcs15-init] pkcs15-lib.c:3555:sc_pkcs15init_update_file: Failed to create file: -1211 (Security status not satisfied)
0x7fc8ab586740 15:01:44.202 [pkcs15-init] pkcs15-lib.c:903:sc_pkcs15init_add_app: returning with: -1211 (Security status not satisfied)
Failed to create PKCS #15 meta structure: Security status not satisfied
0x7fc8ab586740 15:01:44.202 [pkcs15-init] pkcs15-lib.c:417:sc_pkcs15init_unbind: called
0x7fc8ab586740 15:01:44.202 [pkcs15-init] pkcs15-lib.c:418:sc_pkcs15init_unbind: Pksc15init Unbind: 0:(nil):1
0x7fc8ab586740 15:01:44.202 [pkcs15-init] card.c:403:sc_unlock: called
0x7fc8ab586740 15:01:44.202 [pkcs15-init] reader-pcsc.c:566:pcsc_unlock: called
0x7fc8ab586740 15:01:44.209 [pkcs15-init] card.c:305:sc_disconnect_card: called
0x7fc8ab586740 15:01:44.209 [pkcs15-init] reader-pcsc.c:512:pcsc_disconnect: called
0x7fc8ab586740 15:01:44.346 [pkcs15-init] card.c:326:sc_disconnect_card: returning with: 0 (Success)
0x7fc8ab586740 15:01:44.346 [pkcs15-init] ctx.c:815:sc_release_context: called
0x7fc8ab586740 15:01:44.346 [pkcs15-init] reader-pcsc.c:766:pcsc_finish: called
```
I don't know why i got this error and how to fix it. After all i set on the /etc/opensc/opensc.conf the debuging feature but i don't get any debug_file in /tmp/opensc-debug.log to recognize the failure. After all i append -vvv in my creation and i got following logs.

**Comment by frankmorgner** [comment]
Your reader has a PIN-Pad. Could it be that it blocks the PIN verification?

Append `-v` more often and you will see the APDUs. A full log would be more helpfull.

**Comment by frankmorgner** [comment]
Did some previous version of OpenSC erase the card correctly?

**Comment by Nepitwin** [comment]
Yes i use an Card Reader with an Pin Pad exactly I use an ReinerSCT-RFID Smart Card Reader. 

Today I use an older Version from OpenSC Version 0.14 previously I use the latest stable release (0.15).

I Erase my Card with pkcs15-init -E and after all I try it to create an blank card with pkcs15-init --create-pkcs15 --profile pkcs15+onepin --use-default-transport-key --pin 0000 --puk 111111 --label "Starkos2" -vvv

Following Log is Created with this Command usage:

0x7f01990ab740 09:39:17.663 [pkcs15-init] sc.c:231:sc_detect_card_presence: called
0x7f01990ab740 09:39:17.664 [pkcs15-init] reader-pcsc.c:370:pcsc_detect_card_presence: called
0x7f01990ab740 09:39:17.664 [pkcs15-init] reader-pcsc.c:283:refresh_attributes: REINER SCT cyberJack RFID komfort (5654335523) 00 00 check
0x7f01990ab740 09:39:17.664 [pkcs15-init] reader-pcsc.c:307:refresh_attributes: current  state: 0x00000022
0x7f01990ab740 09:39:17.664 [pkcs15-init] reader-pcsc.c:308:refresh_attributes: previous state: 0x00000120
0x7f01990ab740 09:39:17.664 [pkcs15-init] reader-pcsc.c:362:refresh_attributes: card present
0x7f01990ab740 09:39:17.664 [pkcs15-init] reader-pcsc.c:375:pcsc_detect_card_presence: returning with: 1
0x7f01990ab740 09:39:17.664 [pkcs15-init] sc.c:236:sc_detect_card_presence: returning with: 1
Using reader with a card: REINER SCT cyberJack RFID komfort (5654335523) 00 00
0x7f01990ab740 09:39:17.664 [pkcs15-init] sc.c:231:sc_detect_card_presence: called
0x7f01990ab740 09:39:17.664 [pkcs15-init] reader-pcsc.c:370:pcsc_detect_card_presence: called
0x7f01990ab740 09:39:17.664 [pkcs15-init] reader-pcsc.c:283:refresh_attributes: REINER SCT cyberJack RFID komfort (5654335523) 00 00 check
0x7f01990ab740 09:39:17.664 [pkcs15-init] reader-pcsc.c:299:refresh_attributes: returning with: 0 (Success)
0x7f01990ab740 09:39:17.664 [pkcs15-init] reader-pcsc.c:375:pcsc_detect_card_presence: returning with: 1
0x7f01990ab740 09:39:17.664 [pkcs15-init] sc.c:236:sc_detect_card_presence: returning with: 1
0x7f01990ab740 09:39:17.664 [pkcs15-init] card.c:125:sc_connect_card: called
0x7f01990ab740 09:39:17.665 [pkcs15-init] reader-pcsc.c:450:pcsc_connect: called
0x7f01990ab740 09:39:17.665 [pkcs15-init] reader-pcsc.c:283:refresh_attributes: REINER SCT cyberJack RFID komfort (5654335523) 00 00 check
0x7f01990ab740 09:39:17.665 [pkcs15-init] reader-pcsc.c:299:refresh_attributes: returning with: 0 (Success)
0x7f01990ab740 09:39:17.665 [pkcs15-init] reader-pcsc.c:479:pcsc_connect: Initial protocol: T=1
0x7f01990ab740 09:39:17.665 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.665 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:DD:18:00:81:31:FE:45:80:F9:A0:00:00:00:77:01:00:70:0A:90:00:8B
0x7f01990ab740 09:39:17.665 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.665 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.665 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7F:96:00:00:00:31:B9:64:40:70:14:10:73:94:01:80:82:90:00
0x7f01990ab740 09:39:17.665 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.665 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7F:96:00:00:00:31:B8:64:40:70:14:10:73:94:01:80:82:90:00
0x7f01990ab740 09:39:17.665 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:DF:18:FF:81:91:FE:1F:C3:00:31:B8:64:0C:01:EC:C1:73:94:01:80:82:90:00:B3
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:DC:18:FF:81:91:FE:1F:C3:80:73:C8:21:13:66:01:0B:03:52:00:05:38
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:6e:00:ff:45:73:74:45:49:44:20:76:65:72:20:31:2e:30
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:fe:94:00:ff:80:b1:fa:45:1f:03:45:73:74:45:49:44:20:76:65:72:20:31:2e:30:43
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:5e:11:ff:45:73:74:45:49:44:20:76:65:72:20:31:2e:30
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:de:18:ff:c0:80:b1:fe:45:1f:03:45:73:74:45:49:44:20:76:65:72:20:31:2e:30:2b
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.666 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:6e:00:00:45:73:74:45:49:44:20:76:65:72:20:31:2e:30
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:ff:94:00:ff:80:b1:fe:45:1f:03:00:68:d2:76:00:00:28:ff:05:1e:31:80:00:90:00:23
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:ff:11:00:ff:80:b1:fe:45:1f:03:00:68:d2:76:00:00:28:ff:05:1e:31:80:00:90:00:a6
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:146:sc_connect_card: matching configured ATRs
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:155:sc_connect_card: trying driver 'authentic'
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:DD:18:00:81:31:FE:45:80:F9:A0:00:00:00:77:01:00:70:0A:90:00:8B
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:155:sc_connect_card: trying driver 'iasecc'
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.667 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7F:96:00:00:00:31:B9:64:40:70:14:10:73:94:01:80:82:90:00
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7F:96:00:00:00:31:B8:64:40:70:14:10:73:94:01:80:82:90:00
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:DF:18:FF:81:91:FE:1F:C3:00:31:B8:64:0C:01:EC:C1:73:94:01:80:82:90:00:B3
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:DC:18:FF:81:91:FE:1F:C3:80:73:C8:21:13:66:01:0B:03:52:00:05:38
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:185:sc_connect_card: matching built-in ATRs
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:190:sc_connect_card: trying driver 'cardos'
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:e2:00:ff:c1:10:31:fe:55:c8:02:9c
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.668 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:e9:00:ff:c1:10:31:fe:55:00:64:05:00:c8:02:31:80:00:47
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:fb:98:00:ff:c1:10:31:fe:55:00:64:05:20:47:03:31:80:00:90:00:f3
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:fc:98:00:ff:c1:10:31:fe:55:c8:03:49:6e:66:6f:63:61:6d:65:72:65:28
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:f4:98:00:ff:c1:10:31:fe:55:4d:34:63:76:b4
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:f2:18:00:ff:c1:0a:31:fe:55:c8:06:8a
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:d2:18:02:c1:0a:31:fe:58:c8:0d:51
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:d2:18:00:81:31:fe:58:c9:01:14
0x7f01990ab740 09:39:17.669 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:190:sc_connect_card: trying driver 'flex'
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:95:15:40:20:68:01:02:00:00
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:15:40:FF:68:01:02:02:01
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:15:40:FF:68:01:02:02:04
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:85:40:20:68:01:01:05:01
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:94:40:FF:63:01:01:02:01
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.670 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:15:40:FF:63:01:01:02:01
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:18:40:FF:64:02:01:01:02
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:18:40:FF:62:01:01:00:00
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:18:40:FF:62:01:02:01:04
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:18:40:FF:62:04:01:01:05
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:95:15:40:ff:68:01:02:45:47
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:E2:00:00:40:20:49:06
0x7f01990ab740 09:39:17.671 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:E2:00:00:40:20:49:05
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:E2:00:00:40:20:49:07
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:85:40:20:68:01:01:03:05
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:02:14:50
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:19:14:55:90:01:02:01:00:05:04:B0
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:32:15:00:06:80
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.672 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:32:15:00:06:95
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:19:14:59:01:01:0F:01:00:05:08:B0
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:19:14:55:90:01:01:01:00:05:08:B0
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:16:94:81:10:06:01:81:3F
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:16:94:81:10:06:01:81:2F
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:190:sc_connect_card: trying driver 'cyberflex'
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:95:15:40:20:68:01:02:00:00
0x7f01990ab740 09:39:17.673 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.674 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:15:40:FF:68:01:02:02:01
0x7f01990ab740 09:39:17.674 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.674 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:15:40:FF:68:01:02:02:04
0x7f01990ab740 09:39:17.674 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.674 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:85:40:20:68:01:01:05:01
0x7f01990ab740 09:39:17.674 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.674 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:94:40:FF:63:01:01:02:01
0x7f01990ab740 09:39:17.674 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.674 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:15:40:FF:63:01:01:02:01
0x7f01990ab740 09:39:17.674 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.674 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:18:40:FF:64:02:01:01:02
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:18:40:FF:62:01:01:00:00
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:18:40:FF:62:01:02:01:04
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:95:18:40:FF:62:04:01:01:05
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:95:15:40:ff:68:01:02:45:47
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:E2:00:00:40:20:49:06
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:E2:00:00:40:20:49:05
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:E2:00:00:40:20:49:07
0x7f01990ab740 09:39:17.675 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:85:40:20:68:01:01:03:05
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:02:14:50
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:19:14:55:90:01:02:01:00:05:04:B0
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:32:15:00:06:80
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:32:15:00:06:95
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:19:14:59:01:01:0F:01:00:05:08:B0
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:19:14:55:90:01:01:01:00:05:08:B0
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.676 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:16:94:81:10:06:01:81:3F
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:16:94:81:10:06:01:81:2F
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:190:sc_connect_card: trying driver 'gpk'
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:27:00:80:65:A2:04:01:01:37
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:27:00:80:65:A2:05:01:01:37
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:27:00:80:65:A2:0C:01:01:37
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:A7:00:40:14:80:65:A2:14:01:01:37
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:A7:00:40:18:80:65:A2:08:01:01:52
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.677 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:A7:00:40:18:80:65:A2:09:01:01:52
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:A7:00:40:18:80:65:A2:09:01:02:52
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:A7:00:40:18:80:65:A2:09:01:03:52
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:190:sc_connect_card: trying driver 'gemsafeV1'
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7B:94:00:00:80:65:B0:83:01:01:74:83:00:90:00
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:6B:00:00:80:65:B0:83:01:01:74:83:00:90:00
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:6d:00:00:80:31:80:65:b0:83:01:02:90:83:00:90:00
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:6B:00:00:80:65:B0:83:01:03:74:83:00:90:00
0x7f01990ab740 09:39:17.678 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7A:94:00:00:80:65:A2:01:01:01:3D:72:D6:43
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7D:94:00:00:80:31:80:65:B0:83:01:01:90:83:00:90:00
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7D:96:00:00:80:31:80:65:B0:83:11:48:C8:83:00:90:00
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7D:95:00:00:80:31:80:65:B0:83:11:C0:A9:83:00
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7D:95:00:00:80:31:80:65:B0:83:11:C0:A9:83:00:90:00
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7D:95:00:00:80:31:80:65:B0:83:11:00:C8:83:00
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7D:95:00:00:80:31:80:65:B0:83:11:00:C8:83:00:90:00
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:7D:96:00:00:80:31:80:65:B0:83:11:00:C8:83:00:90:00
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.679 [pkcs15-init] card.c:190:sc_connect_card: trying driver 'miocos'
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:9D:94:40:23:00:68:10:11:4D:69:6F:43:4F:53:00:90:00
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:9D:94:40:23:00:68:20:01:4D:69:6F:43:4F:53:00:90:00
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:190:sc_connect_card: trying driver 'mcrd'
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:FF:94:00:FF:80:B1:FE:45:1F:03:00:68:D2:76:00:00:28:FF:05:1E:31:80:00:90:00:23
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:6f:00:ff:00:68:d2:76:00:00:28:ff:05:1e:31:80:00:90:00
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:ff:11:00:ff:80:b1:fe:45:1f:03:00:68:d2:76:00:00:28:ff:05:1e:31:80:00:90:00:a6
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:FE:94:00:FF:80:B1:FA:45:1F:03:45:73:74:45:49:44:20
0x7f01990ab740 09:39:17.680 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:fe:94:00:ff:80:b1:fa:45:1f:03:45:73:74:45:49:44:20:76:65:72:20:31:2e:30:43
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:6e:00:ff:45:73:74:45:49:44:20:76:65:72:20:31:2e:30
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:de:18:ff:c0:80:b1:fe:45:1f:03:45:73:74:45:49:44:20:76:65:72:20:31:2e:30:2b
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:5e:11:ff:45:73:74:45:49:44:20:76:65:72:20:31:2e:30
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:6e:00:00:45:73:74:45:49:44:20:76:65:72:20:31:2e:30
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:FE:18:00:00:80:31:FE:45:45:73:74:45:49:44:20:76:65:72:20:31:2E:30:A8
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:FE:18:00:00:80:31:FE:45:80:31:80:66:40:90:A4:56:1B:16:83:01:90:00:86
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.681 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:fe:18:00:00:80:31:fe:45:80:31:80:66:40:90:a4:16:2a:00:83:01:90:00:e1
0x7f01990ab740 09:39:17.682 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.682 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:fe:18:00:00:80:31:fe:45:80:31:80:66:40:90:a4:16:2a:00:83:0f:90:00:ef
0x7f01990ab740 09:39:17.682 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.682 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:fa:18:00:00:80:31:fe:45:fe:65:49:44:20:2f:20:50:4b:49:03
0x7f01990ab740 09:39:17.682 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.682 [pkcs15-init] apdu.c:559:sc_transmit_apdu: called
0x7f01990ab740 09:39:17.682 [pkcs15-init] card.c:325:sc_lock: called
0x7f01990ab740 09:39:17.682 [pkcs15-init] reader-pcsc.c:517:pcsc_lock: called
0x7f01990ab740 09:39:17.682 [pkcs15-init] apdu.c:526:sc_transmit: called
0x7f01990ab740 09:39:17.682 [pkcs15-init] apdu.c:380:sc_single_transmit: called
0x7f01990ab740 09:39:17.682 [pkcs15-init] apdu.c:385:sc_single_transmit: CLA:0, INS:A4, P1:4, P2:0, data(15) 0x7f0198ad1890
0x7f01990ab740 09:39:17.682 [pkcs15-init] reader-pcsc.c:249:pcsc_transmit: reader 'REINER SCT cyberJack RFID komfort (5654335523) 00 00'
0x7f01990ab740 09:39:17.682 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Outgoing APDU data [   20 bytes] =====================================
00 A4 04 00 0F D2 33 00 00 00 45 73 74 45 49 44 ......3...EstEID
20 76 33 35                                      v35
======================================================================
0x7f01990ab740 09:39:17.682 [pkcs15-init] reader-pcsc.c:182:pcsc_internal_transmit: called
0x7f01990ab740 09:39:17.906 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Incoming APDU data [    2 bytes] =====================================
6A 82 j.
======================================================================
0x7f01990ab740 09:39:17.906 [pkcs15-init] apdu.c:395:sc_single_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.906 [pkcs15-init] apdu.c:548:sc_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.906 [pkcs15-init] card.c:361:sc_unlock: called
0x7f01990ab740 09:39:17.906 [pkcs15-init] reader-pcsc.c:554:pcsc_unlock: called
0x7f01990ab740 09:39:17.914 [pkcs15-init] card-mcrd.c:292:mcrd_match_card: SELECT AID: 6A82
0x7f01990ab740 09:39:17.914 [pkcs15-init] card.c:190:sc_connect_card: trying driver 'asepcos'
0x7f01990ab740 09:39:17.914 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.914 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:d6:18:00:81:b1:80:7d:1f:03:80:51:00:61:10:30:8f
0x7f01990ab740 09:39:17.914 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.914 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3b:d6:18:00:81:b1:fe:7d:1f:03:41:53:45:37:35:35:01
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:190:sc_connect_card: trying driver 'starcos'
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:878:match_atr_table: ATR     : 3b:b7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:B7:94:00:c0:24:31:fe:65:53:50:4b:32:33:90:00:b4
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:892:match_atr_table: ignored - wrong length
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:889:match_atr_table: ATR try : 3B:B7:94:00:81:31:fe:65:53:50:4b:32:33:90:00:d1
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:203:sc_connect_card: matched: STARCOS SPK 2.3/2.4
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:239:sc_connect_card: card info name:'STARCOS SPK 2.3', type:7001, flags:0x0, max_send/recv_size:128/128
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:1220:sc_card_sm_check: called
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:1221:sc_card_sm_check: card->sm_ctx.ops.open (nil)
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:1226:sc_card_sm_check: returning with: 0 (Success)
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:250:sc_connect_card: returning with: 0 (Success)
0x7f01990ab740 09:39:17.915 [pkcs15-init] card.c:325:sc_lock: called
0x7f01990ab740 09:39:17.915 [pkcs15-init] reader-pcsc.c:517:pcsc_lock: called
0x7f01990ab740 09:39:17.916 [pkcs15-init] pkcs15-lib.c:299:sc_pkcs15init_bind: called
0x7f01990ab740 09:39:17.916 [pkcs15-init] card.c:769:sc_card_ctl: called
0x7f01990ab740 09:39:17.916 [pkcs15-init] card.c:776:sc_card_ctl: card_ctl(4) not supported
0x7f01990ab740 09:39:17.916 [pkcs15-init] card.c:618:sc_select_file: called; type=2, path=3f0050154946
0x7f01990ab740 09:39:17.916 [pkcs15-init] card-starcos.c:368:starcos_select_file: called
0x7f01990ab740 09:39:17.916 [pkcs15-init] card-starcos.c:378:starcos_select_file: current path (path, valid):  (len: 0)
0x7f01990ab740 09:39:17.916 [pkcs15-init] apdu.c:559:sc_transmit_apdu: called
0x7f01990ab740 09:39:17.916 [pkcs15-init] card.c:325:sc_lock: called
0x7f01990ab740 09:39:17.916 [pkcs15-init] apdu.c:526:sc_transmit: called
0x7f01990ab740 09:39:17.916 [pkcs15-init] apdu.c:380:sc_single_transmit: called
0x7f01990ab740 09:39:17.916 [pkcs15-init] apdu.c:385:sc_single_transmit: CLA:0, INS:A4, P1:0, P2:0, data(2) 0x7ffffda90c70
0x7f01990ab740 09:39:17.916 [pkcs15-init] reader-pcsc.c:249:pcsc_transmit: reader 'REINER SCT cyberJack RFID komfort (5654335523) 00 00'
0x7f01990ab740 09:39:17.916 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Outgoing APDU data [    8 bytes] =====================================
00 A4 00 00 02 3F 00 00 .....?..
======================================================================
0x7f01990ab740 09:39:17.916 [pkcs15-init] reader-pcsc.c:182:pcsc_internal_transmit: called
0x7f01990ab740 09:39:17.922 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Incoming APDU data [    2 bytes] =====================================
62 84 b.
======================================================================
0x7f01990ab740 09:39:17.922 [pkcs15-init] apdu.c:395:sc_single_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.922 [pkcs15-init] apdu.c:548:sc_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.922 [pkcs15-init] card.c:361:sc_unlock: called
0x7f01990ab740 09:39:17.922 [pkcs15-init] apdu.c:559:sc_transmit_apdu: called
0x7f01990ab740 09:39:17.922 [pkcs15-init] card.c:325:sc_lock: called
0x7f01990ab740 09:39:17.922 [pkcs15-init] apdu.c:526:sc_transmit: called
0x7f01990ab740 09:39:17.922 [pkcs15-init] apdu.c:380:sc_single_transmit: called
0x7f01990ab740 09:39:17.922 [pkcs15-init] apdu.c:385:sc_single_transmit: CLA:0, INS:A4, P1:0, P2:C, data(2) 0x7ffffda90c70
0x7f01990ab740 09:39:17.923 [pkcs15-init] reader-pcsc.c:249:pcsc_transmit: reader 'REINER SCT cyberJack RFID komfort (5654335523) 00 00'
0x7f01990ab740 09:39:17.923 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Outgoing APDU data [    7 bytes] =====================================
00 A4 00 0C 02 3F 00 .....?.
======================================================================
0x7f01990ab740 09:39:17.923 [pkcs15-init] reader-pcsc.c:182:pcsc_internal_transmit: called
0x7f01990ab740 09:39:17.928 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Incoming APDU data [    2 bytes] =====================================
90 00 ..
======================================================================
0x7f01990ab740 09:39:17.928 [pkcs15-init] apdu.c:395:sc_single_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.928 [pkcs15-init] apdu.c:548:sc_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.928 [pkcs15-init] card.c:361:sc_unlock: called
0x7f01990ab740 09:39:17.928 [pkcs15-init] card-starcos.c:356:starcos_select_fid: returning with: 0 (Success)
0x7f01990ab740 09:39:17.928 [pkcs15-init] apdu.c:559:sc_transmit_apdu: called
0x7f01990ab740 09:39:17.928 [pkcs15-init] card.c:325:sc_lock: called
0x7f01990ab740 09:39:17.928 [pkcs15-init] apdu.c:526:sc_transmit: called
0x7f01990ab740 09:39:17.928 [pkcs15-init] apdu.c:380:sc_single_transmit: called
0x7f01990ab740 09:39:17.928 [pkcs15-init] apdu.c:385:sc_single_transmit: CLA:0, INS:A4, P1:0, P2:0, data(2) 0x7ffffda90c70
0x7f01990ab740 09:39:17.929 [pkcs15-init] reader-pcsc.c:249:pcsc_transmit: reader 'REINER SCT cyberJack RFID komfort (5654335523) 00 00'
0x7f01990ab740 09:39:17.929 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Outgoing APDU data [    8 bytes] =====================================
00 A4 00 00 02 50 15 00 .....P..
======================================================================
0x7f01990ab740 09:39:17.929 [pkcs15-init] reader-pcsc.c:182:pcsc_internal_transmit: called
0x7f01990ab740 09:39:17.934 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Incoming APDU data [    2 bytes] =====================================
62 84 b.
======================================================================
0x7f01990ab740 09:39:17.934 [pkcs15-init] apdu.c:395:sc_single_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.934 [pkcs15-init] apdu.c:548:sc_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.935 [pkcs15-init] card.c:361:sc_unlock: called
0x7f01990ab740 09:39:17.935 [pkcs15-init] apdu.c:559:sc_transmit_apdu: called
0x7f01990ab740 09:39:17.935 [pkcs15-init] card.c:325:sc_lock: called
0x7f01990ab740 09:39:17.935 [pkcs15-init] apdu.c:526:sc_transmit: called
0x7f01990ab740 09:39:17.935 [pkcs15-init] apdu.c:380:sc_single_transmit: called
0x7f01990ab740 09:39:17.935 [pkcs15-init] apdu.c:385:sc_single_transmit: CLA:0, INS:A4, P1:0, P2:C, data(2) 0x7ffffda90c70
0x7f01990ab740 09:39:17.935 [pkcs15-init] reader-pcsc.c:249:pcsc_transmit: reader 'REINER SCT cyberJack RFID komfort (5654335523) 00 00'
0x7f01990ab740 09:39:17.935 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Outgoing APDU data [    7 bytes] =====================================
00 A4 00 0C 02 50 15 .....P.
======================================================================
0x7f01990ab740 09:39:17.935 [pkcs15-init] reader-pcsc.c:182:pcsc_internal_transmit: called
0x7f01990ab740 09:39:17.940 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Incoming APDU data [    2 bytes] =====================================
90 00 ..
======================================================================
0x7f01990ab740 09:39:17.940 [pkcs15-init] apdu.c:395:sc_single_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.940 [pkcs15-init] apdu.c:548:sc_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.940 [pkcs15-init] card.c:361:sc_unlock: called
0x7f01990ab740 09:39:17.940 [pkcs15-init] card-starcos.c:356:starcos_select_fid: returning with: 0 (Success)
0x7f01990ab740 09:39:17.941 [pkcs15-init] apdu.c:559:sc_transmit_apdu: called
0x7f01990ab740 09:39:17.941 [pkcs15-init] card.c:325:sc_lock: called
0x7f01990ab740 09:39:17.941 [pkcs15-init] apdu.c:526:sc_transmit: called
0x7f01990ab740 09:39:17.941 [pkcs15-init] apdu.c:380:sc_single_transmit: called
0x7f01990ab740 09:39:17.941 [pkcs15-init] apdu.c:385:sc_single_transmit: CLA:0, INS:A4, P1:0, P2:0, data(2) 0x7ffffda90c70
0x7f01990ab740 09:39:17.941 [pkcs15-init] reader-pcsc.c:249:pcsc_transmit: reader 'REINER SCT cyberJack RFID komfort (5654335523) 00 00'
0x7f01990ab740 09:39:17.941 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Outgoing APDU data [    8 bytes] =====================================
00 A4 00 00 02 49 46 00 .....IF.
======================================================================
0x7f01990ab740 09:39:17.941 [pkcs15-init] reader-pcsc.c:182:pcsc_internal_transmit: called
0x7f01990ab740 09:39:17.946 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Incoming APDU data [    2 bytes] =====================================
6A 82 j.
======================================================================
0x7f01990ab740 09:39:17.946 [pkcs15-init] apdu.c:395:sc_single_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.946 [pkcs15-init] apdu.c:548:sc_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.946 [pkcs15-init] card.c:361:sc_unlock: called
0x7f01990ab740 09:39:17.946 [pkcs15-init] card-starcos.c:1232:starcos_check_sw: sw1 = 0x6a, sw2 = 0x82
0x7f01990ab740 09:39:17.946 [pkcs15-init] iso7816.c:102:iso7816_check_sw: File not found
0x7f01990ab740 09:39:17.946 [pkcs15-init] card-starcos.c:312:starcos_select_fid: returning with: -1201 (File not found)
0x7f01990ab740 09:39:17.946 [pkcs15-init] card.c:640:sc_select_file: 'SELECT' error: -1201 (File not found)
0x7f01990ab740 09:39:17.946 [pkcs15-init] profile.c:338:sc_profile_load: called
0x7f01990ab740 09:39:17.946 [pkcs15-init] profile.c:368:sc_profile_load: Using profile directory '/usr/share/opensc'.
0x7f01990ab740 09:39:17.946 [pkcs15-init] profile.c:376:sc_profile_load: Trying profile file /usr/share/opensc/pkcs15.profile
0x7f01990ab740 09:39:17.947 [pkcs15-init] profile.c:381:sc_profile_load: profile /usr/share/opensc/pkcs15.profile loaded ok
0x7f01990ab740 09:39:17.947 [pkcs15-init] profile.c:391:sc_profile_load: returning with: 0 (Success)
0x7f01990ab740 09:39:17.947 [pkcs15-init] profile.c:338:sc_profile_load: called
0x7f01990ab740 09:39:17.947 [pkcs15-init] profile.c:368:sc_profile_load: Using profile directory '/usr/share/opensc'.
0x7f01990ab740 09:39:17.947 [pkcs15-init] profile.c:376:sc_profile_load: Trying profile file /usr/share/opensc/starcos.profile
0x7f01990ab740 09:39:17.947 [pkcs15-init] profile.c:381:sc_profile_load: profile /usr/share/opensc/starcos.profile loaded ok
0x7f01990ab740 09:39:17.948 [pkcs15-init] profile.c:391:sc_profile_load: returning with: 0 (Success)
0x7f01990ab740 09:39:17.948 [pkcs15-init] profile.c:403:sc_profile_finish: called
0x7f01990ab740 09:39:17.948 [pkcs15-init] profile.c:446:sc_profile_finish: returning with: 0 (Success)
0x7f01990ab740 09:39:17.948 [pkcs15-init] pkcs15-lib.c:406:sc_pkcs15init_bind: returning with: 0 (Success)
0x7f01990ab740 09:39:17.948 [pkcs15-init] pkcs15-lib.c:764:sc_pkcs15init_add_app: called
0x7f01990ab740 09:39:17.948 [pkcs15-init] pkcs15-lib.c:3730:sc_pkcs15init_qualify_pin: called
0x7f01990ab740 09:39:17.948 [pkcs15-init] pkcs15-lib.c:3745:sc_pkcs15init_qualify_pin: returning with: 0 (Success)
0x7f01990ab740 09:39:17.948 [pkcs15-init] pkcs15-lib.c:3730:sc_pkcs15init_qualify_pin: called
0x7f01990ab740 09:39:17.948 [pkcs15-init] pkcs15-lib.c:3745:sc_pkcs15init_qualify_pin: returning with: 0 (Success)
0x7f01990ab740 09:39:17.948 [pkcs15-init] pkcs15-lib.c:816:sc_pkcs15init_add_app: Add virtual SO_PIN('User PIN',flags:30,reference:1,path:'')
0x7f01990ab740 09:39:17.948 [pkcs15-init] card.c:618:sc_select_file: called; type=2, path=3f00
0x7f01990ab740 09:39:17.948 [pkcs15-init] card-starcos.c:368:starcos_select_file: called
0x7f01990ab740 09:39:17.948 [pkcs15-init] card-starcos.c:378:starcos_select_file: current path (path, valid): 3f005015 (len: 4)
0x7f01990ab740 09:39:17.948 [pkcs15-init] apdu.c:559:sc_transmit_apdu: called
0x7f01990ab740 09:39:17.948 [pkcs15-init] card.c:325:sc_lock: called
0x7f01990ab740 09:39:17.948 [pkcs15-init] apdu.c:526:sc_transmit: called
0x7f01990ab740 09:39:17.948 [pkcs15-init] apdu.c:380:sc_single_transmit: called
0x7f01990ab740 09:39:17.948 [pkcs15-init] apdu.c:385:sc_single_transmit: CLA:0, INS:A4, P1:0, P2:0, data(2) 0x7ffffda91080
0x7f01990ab740 09:39:17.948 [pkcs15-init] reader-pcsc.c:249:pcsc_transmit: reader 'REINER SCT cyberJack RFID komfort (5654335523) 00 00'
0x7f01990ab740 09:39:17.949 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Outgoing APDU data [    8 bytes] =====================================
00 A4 00 00 02 3F 00 00 .....?..
======================================================================
0x7f01990ab740 09:39:17.949 [pkcs15-init] reader-pcsc.c:182:pcsc_internal_transmit: called
0x7f01990ab740 09:39:17.954 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Incoming APDU data [    2 bytes] =====================================
62 84 b.
======================================================================
0x7f01990ab740 09:39:17.954 [pkcs15-init] apdu.c:395:sc_single_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.954 [pkcs15-init] apdu.c:548:sc_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.954 [pkcs15-init] card.c:361:sc_unlock: called
0x7f01990ab740 09:39:17.954 [pkcs15-init] apdu.c:559:sc_transmit_apdu: called
0x7f01990ab740 09:39:17.954 [pkcs15-init] card.c:325:sc_lock: called
0x7f01990ab740 09:39:17.954 [pkcs15-init] apdu.c:526:sc_transmit: called
0x7f01990ab740 09:39:17.954 [pkcs15-init] apdu.c:380:sc_single_transmit: called
0x7f01990ab740 09:39:17.954 [pkcs15-init] apdu.c:385:sc_single_transmit: CLA:0, INS:A4, P1:0, P2:C, data(2) 0x7ffffda91080
0x7f01990ab740 09:39:17.955 [pkcs15-init] reader-pcsc.c:249:pcsc_transmit: reader 'REINER SCT cyberJack RFID komfort (5654335523) 00 00'
0x7f01990ab740 09:39:17.955 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Outgoing APDU data [    7 bytes] =====================================
00 A4 00 0C 02 3F 00 .....?.
======================================================================
0x7f01990ab740 09:39:17.955 [pkcs15-init] reader-pcsc.c:182:pcsc_internal_transmit: called
0x7f01990ab740 09:39:17.960 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Incoming APDU data [    2 bytes] =====================================
90 00 ..
======================================================================
0x7f01990ab740 09:39:17.960 [pkcs15-init] apdu.c:395:sc_single_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.960 [pkcs15-init] apdu.c:548:sc_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.960 [pkcs15-init] card.c:361:sc_unlock: called
0x7f01990ab740 09:39:17.960 [pkcs15-init] card-starcos.c:356:starcos_select_fid: returning with: 0 (Success)
0x7f01990ab740 09:39:17.960 [pkcs15-init] card.c:646:sc_select_file: returning with: 0 (Success)
0x7f01990ab740 09:39:17.960 [pkcs15-init] card.c:769:sc_card_ctl: called
0x7f01990ab740 09:39:17.960 [pkcs15-init] card-starcos.c:713:starcos_create_df: creating DF
0x7f01990ab740 09:39:17.960 [pkcs15-init] card-starcos.c:715:starcos_create_df: calling REGISTER DF
0x7f01990ab740 09:39:17.960 [pkcs15-init] apdu.c:559:sc_transmit_apdu: called
0x7f01990ab740 09:39:17.960 [pkcs15-init] card.c:325:sc_lock: called
0x7f01990ab740 09:39:17.960 [pkcs15-init] apdu.c:526:sc_transmit: called
0x7f01990ab740 09:39:17.960 [pkcs15-init] apdu.c:380:sc_single_transmit: called
0x7f01990ab740 09:39:17.960 [pkcs15-init] apdu.c:385:sc_single_transmit: CLA:80, INS:52, P1:3E, P2:80, data(15) 0x7ffffda914e4
0x7f01990ab740 09:39:17.960 [pkcs15-init] reader-pcsc.c:249:pcsc_transmit: reader 'REINER SCT cyberJack RFID komfort (5654335523) 00 00'
0x7f01990ab740 09:39:17.961 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Outgoing APDU data [   20 bytes] =====================================
80 52 3E 80 0F 50 15 0C A0 00 00 00 63 50 4B 43 .R>..P......cPKC
53 2D 31 35                                     S-15
======================================================================
0x7f01990ab740 09:39:17.961 [pkcs15-init] reader-pcsc.c:182:pcsc_internal_transmit: called
0x7f01990ab740 09:39:17.967 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Incoming APDU data [    2 bytes] =====================================
6A 00 j.
======================================================================
0x7f01990ab740 09:39:17.967 [pkcs15-init] apdu.c:395:sc_single_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.967 [pkcs15-init] apdu.c:548:sc_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.967 [pkcs15-init] card.c:361:sc_unlock: called
0x7f01990ab740 09:39:17.968 [pkcs15-init] card-starcos.c:728:starcos_create_df: calling CREATE DF
0x7f01990ab740 09:39:17.968 [pkcs15-init] apdu.c:559:sc_transmit_apdu: called
0x7f01990ab740 09:39:17.968 [pkcs15-init] card.c:325:sc_lock: called
0x7f01990ab740 09:39:17.968 [pkcs15-init] apdu.c:526:sc_transmit: called
0x7f01990ab740 09:39:17.968 [pkcs15-init] apdu.c:380:sc_single_transmit: called
0x7f01990ab740 09:39:17.968 [pkcs15-init] apdu.c:385:sc_single_transmit: CLA:80, INS:E0, P1:1, P2:0, data(25) 0x7ffffda914e4
0x7f01990ab740 09:39:17.968 [pkcs15-init] reader-pcsc.c:249:pcsc_transmit: reader 'REINER SCT cyberJack RFID komfort (5654335523) 00 00'
0x7f01990ab740 09:39:17.968 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Outgoing APDU data [   30 bytes] =====================================
80 E0 01 00 19 50 15 0C A0 00 00 00 63 50 4B 43 .....P......cPKC
53 2D 31 35 00 00 00 00 0A 00 01 01 00 00       S-15..........
======================================================================
0x7f01990ab740 09:39:17.968 [pkcs15-init] reader-pcsc.c:182:pcsc_internal_transmit: called
0x7f01990ab740 09:39:17.976 [pkcs15-init] apdu.c:185:sc_apdu_log: 
Incoming APDU data [    2 bytes] =====================================
69 82 i.
======================================================================
0x7f01990ab740 09:39:17.976 [pkcs15-init] apdu.c:395:sc_single_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.977 [pkcs15-init] apdu.c:548:sc_transmit: returning with: 0 (Success)
0x7f01990ab740 09:39:17.977 [pkcs15-init] card.c:361:sc_unlock: called
0x7f01990ab740 09:39:17.977 [pkcs15-init] card-starcos.c:1232:starcos_check_sw: sw1 = 0x69, sw2 = 0x82
0x7f01990ab740 09:39:17.977 [pkcs15-init] iso7816.c:102:iso7816_check_sw: Security status not satisfied
0x7f01990ab740 09:39:17.977 [pkcs15-init] card.c:779:sc_card_ctl: returning with: -1211 (Security status not satisfied)
0x7f01990ab740 09:39:17.977 [pkcs15-init] pkcs15-lib.c:834:sc_pkcs15init_add_app: Create 'DIR' error: -1211 (Security status not satisfied)
Failed to create PKCS #15 meta structure: Security status not satisfied
0x7f01990ab740 09:39:17.977 [pkcs15-init] pkcs15-lib.c:416:sc_pkcs15init_unbind: called
0x7f01990ab740 09:39:17.977 [pkcs15-init] pkcs15-lib.c:417:sc_pkcs15init_unbind: Pksc15init Unbind: 0:(nil):1
0x7f01990ab740 09:39:17.977 [pkcs15-init] card.c:361:sc_unlock: called
0x7f01990ab740 09:39:17.977 [pkcs15-init] reader-pcsc.c:554:pcsc_unlock: called
0x7f01990ab740 09:39:17.982 [pkcs15-init] card.c:267:sc_disconnect_card: called
0x7f01990ab740 09:39:17.982 [pkcs15-init] reader-pcsc.c:504:pcsc_disconnect: called
0x7f01990ab740 09:39:18.120 [pkcs15-init] card.c:288:sc_disconnect_card: returning with: 0 (Success)
0x7f01990ab740 09:39:18.120 [pkcs15-init] ctx.c:795:sc_release_context: called
0x7f01990ab740 09:39:18.120 [pkcs15-init] reader-pcsc.c:745:pcsc_finish: called
Connecting to card in reader REINER SCT cyberJack RFID komfort (5654335523) 00 00...
Using card driver STARCOS SPK 2.3/2.4.
About to create PKCS #15 meta structure.

**Comment by frankmorgner** [comment]
If I understand correctly, this problem also existed in previous version of OpenSC. I can't really reproduce this problem since I am missing the Starcos card... Also, I am not sure if this *should* work or if it is tested in any way!

**Comment by frankmorgner** [comment]
Since there has not been much progress and there is nobody working on this issue. I will close it as won't fix. Please re-open this PR if you have new input if if someone wants to fix this problem.
